0008-atomic-registry.md

```diff
@@ -0,0 +1,144 @@
+- Feature Name: Katello and Atomic Registry Integration
+- Start Date: 2016-08-11
+- RFC PR:
+- Issue:
+
+Table of Contents
+ * [Summary](#summary)
+ * [Benefits](#benefits)
+ * [Example](#example)
+ * [User Authentication and Role Based Access Control](#userauthentication)
+ * [Installation](#installation)
+ * [Capsules](#capsules)
+
+# Summary
+[summary]: #summary
+
+To facilitate a more natural feel for working with containers and [Katello](https://katello.org), [Atomic Registry](http://www.projectatomic.io/registry/) will become an optional integration point. This true registry will provide the full set of features required by users, while allowing Katello to continue to act as a container repository. Integration between the two will allow images in the Katello content management lifecycle to be directly linked to images in Atomic Registry projects, and vice versa.
```

Comment:
I think the 'true registry' statement leads to confusion so early without a one sentence explanation of the background or removal of it all together.